styles.css

```diff
@@ -0,0 +1,1108 @@
+
+.python_lecturer,
+.RR_lecturer,
+.AND_lecturer,
+.CS_lecturer,
+.Dean_lecturer,
+.IA_lecturer {
+  display: flex;
+  gap: 15px;
+  margin: 0 20px 10px 20px;
+  padding: 20px;
+}
+
+.Java_Lecturer {
+  display: flex;
+  gap: 15px;
+  justify-content: space-between;
+  margin: 0 20px 10px 20px;
+  padding: 20px;
+}
+
+.more_lecturers {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid #5a5a5a;
+  border-radius: 3px;
+  padding: 0.7rem 0;
+  margin: 1rem 0 2rem 1rem;
+  width: 90%;
+  gap: 0.5rem;
+}
+
+.down {
+  width: 16px;
+  height: 14px;
+  margin-left: 5px;
+}
+
+.more_info:hover {
+  background-color: var(--COLOR2);
+  cursor: pointer;
+}
+
+.more_info:active {
+  background-color: red;
+  cursor: pointer;
+}
+
+.contact {
+  background-color: rgb(244, 244, 244);
+}
+
+.contact_orbit {
+  font-size: 30px;
+  line-height: 52px;
+  text-align: center;
+  color: var(--COLOR2);
+  margin-bottom: 12px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+
+.list {
+  flex-direction: column;
+}
+
+input {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 300px;
+  height: 40px;
+  margin-top: 15px;
+}
+
+textarea {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 300px;
+  height: 150px;
+  margin-top: 15px;
+}
+
+.contact-btn {
+  font-family: var(--FF);
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.03em;
+  width: 150px;
+  height: 50px;
+  color: var(--COLOR2);
+  border-radius: 8px;
+  margin-top: 10px;
+}
+
+hr {
+  background-color: #ec5242;
+  border: 0.2px solid #ec5242;
+  border-radius: 2px;
+  width: 35px;
+}
+
+#hr-feature,
+#hr-partner,
+#hr-orbit,
+#hr-alumni,
+#partner-hr {
+  margin-top: -32px;
+}
+
+#logo-hr {
+  width: 100%;
+  border: 0.1px solid lightgray;
+  margin-top: 10px;
+}
+
+.contact-btn:hover {
+  background-color: var(--COLOR1);
+  cursor: pointer;
+}
+
+.contact-btn:active {
+  background-color: red;
+  cursor: pointer;
+}
+
+.orbit {
+  text-align: center;
+  padding: 40px;
+  margin: 10px;
+
+  /* position: relative; */
+  font-family: var(--FF);
+}
+
+.logo_orbit {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.gallery div {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(240 80 50 / 75%);
+  transition: all 0.3s ease;
+  padding: 4.5rem 0;
+  cursor: pointer;
+  gap: 0.3rem;
+}
+
+.orbit_des {
+  padding: 10px;
+  text-align: center;
+  margin: 10px;
+  font-family: var(--FF);
+}
+
+.logo_im {
+  width: 320px;
+  height: 130px;
+  margin: 45px;
+  border: 1px solid grey;
+  padding: 5px;
+  align-self: center;
+}
+
+.alumni {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  /* padding: 2.5rem;
+    gap: 1rem; */
```

Comment:
- [ ] It is not a good practice to leave unused code as a comment. It would be great if you remove them to make the codebase more clean and professional. 🙂️